src/plugins/plot/MctTicks.vue

```diff
@@ -132,6 +147,10 @@ export default {
     this.updateTicks();
   },
   beforeUnmount() {
+    this.removeAlignment({
+      yAxisId: this.axisId,
+      updateObjectPath: this.path
+    });
```

Comment:
we should move this to a `onBeforeUnmount()` hook in the composable itself. Composables reduce the need for boilerplate in components by encapsulating the hook logic entirely within the composable.

Reply:
See comment above. The removing of alignment is specific to the MctTicks component and requires the axisId to be passed in. Maybe I don't completely understand how this would work from within a composable.